winterfell/services/fileClaimService.js

```diff
@@ -8,39 +8,28 @@ function FileClaimService(app) {
 };
 
 module.exports = FileClaimService;
-module.exports.createNewClaim = function(userId, callbacks) {
-  FileClaim.find({}).exec(function(err, claims) {
-    console.log('claimsAll: ' + JSON.stringify(claims));
-  });
+module.exports.createNewClaim = function(userId, callback) {
   return FileClaim.findOne({ userId: userId, state: FileClaim.State.INCOMPLETE }).exec(function(err, fileClaim) {
-    console.log('claims: ' + JSON.stringify(fileClaim));
     if (_.isEmpty(fileClaim)) {
-      return FileClaim.quickCreate(userId).then(function(claim, error) {
-        if (claim) {
-          _.isEmpty(callbacks) ? null : callbacks.onSuccess(claim);
-          return claim;
-        } else {
-          _.isEmpty(callbacks) ? null : callbacks.onError(http.INTERNAL_SERVER_ERROR, httpErrors.DATABASE);
-          return null;
-        }
-      });
-    } else {
-      _.isEmpty(callbacks) ? null : callbacks.onError(http.BAD_REQUEST, httpErrors.CLAIM_INCOMPLETE_EXISTS);
+      return FileClaim.quickCreate(userId, callback);
+    }
+    if (_.isFunction(callback)) {
```

Comment:
how I've been doing callbacks is invoke a success if the expected object is defined (in this case `fileClaim` is not `undefined`) and invoke the error callback otherwise.

I had the `_.isFunction()` just to be more type-safe and to invoke the error callback, making the second argument `undefined`.